src/Hal/Serializer/ItemNormalizer.php

```diff
@@ -232,4 +240,36 @@ private function getHalCacheKey(string $format = null, array $context)
             return false;
         }
     }
+
+    /**
+     * Is the max depth reached for the given attribute?
+     *
+     * @param AttributeMetadataInterface[] $attributesMetadata
+     */
+    private function isMaxDepthReached(array $attributesMetadata, string $class, string $attribute, array &$context): bool
+    {
+        if (
+            !isset($context[static::ENABLE_MAX_DEPTH]) ||
```

Comment:
Can be `($context[static::ENABLE_MAX_DEPTH] ?? false)` (not possible in Symfony because we still support PHP 5)/

Reply:
almost, it's `!($context[static::ENABLE_MAX_DEPTH] ?? false)`.
done